src/core/src/service_interfaces/StatusHandler.py

```diff
@@ -247,16 +260,25 @@ def set_installation_reboot_status(self, new_reboot_status):
-        if (new_reboot_status == Constants.RebootStatus.NOT_NEEDED and self.__installation_reboot_status not in [Constants.RebootStatus.NOT_NEEDED])\
-                or (new_reboot_status == Constants.RebootStatus.REQUIRED and self.__installation_reboot_status not in [Constants.RebootStatus.NOT_NEEDED, Constants.RebootStatus.REQUIRED, Constants.RebootStatus.COMPLETED])\
-                or (new_reboot_status == Constants.RebootStatus.STARTED and self.__installation_reboot_status not in [Constants.RebootStatus.NOT_NEEDED, Constants.RebootStatus.REQUIRED, Constants.RebootStatus.STARTED])\
-                or (new_reboot_status == Constants.RebootStatus.FAILED and self.__installation_reboot_status not in [Constants.RebootStatus.STARTED, Constants.RebootStatus.FAILED])\
-                or (new_reboot_status == Constants.RebootStatus.COMPLETED and self.__installation_reboot_status not in [Constants.RebootStatus.STARTED, Constants.RebootStatus.COMPLETED]):
-            self.composite_logger.log_error("Invalid reboot status transition attempted. [CurrentRebootStatus={0}] [NewRebootStatus={1}]".format(self.__installation_reboot_status, str(new_reboot_status)))
+        if (new_reboot_status == Constants.RebootStatus.NOT_NEEDED and self.__installation_reboot_status not in [
+            Constants.RebootStatus.NOT_NEEDED]) \
+                or (new_reboot_status == Constants.RebootStatus.REQUIRED and self.__installation_reboot_status not in [
+            Constants.RebootStatus.NOT_NEEDED, Constants.RebootStatus.REQUIRED, Constants.RebootStatus.COMPLETED]) \
+                or (new_reboot_status == Constants.RebootStatus.STARTED and self.__installation_reboot_status not in [
+            Constants.RebootStatus.NOT_NEEDED, Constants.RebootStatus.REQUIRED, Constants.RebootStatus.STARTED]) \
+                or (new_reboot_status == Constants.RebootStatus.FAILED and self.__installation_reboot_status not in [
+            Constants.RebootStatus.STARTED, Constants.RebootStatus.FAILED]) \
+                or (new_reboot_status == Constants.RebootStatus.COMPLETED and self.__installation_reboot_status not in [
+            Constants.RebootStatus.STARTED, Constants.RebootStatus.COMPLETED]):
+            self.composite_logger.log_error(
+                "Invalid reboot status transition attempted. [CurrentRebootStatus={0}] [NewRebootStatus={1}]".format(
+                    self.__installation_reboot_status, str(new_reboot_status)))
             return
 
         # Persisting new reboot status (with machine state incorporation)
-        self.composite_logger.log_debug("Setting new installation reboot status. [NewRebootStatus={0}] [CurrentRebootStatus={1}]".format(str(new_reboot_status), self.__installation_reboot_status))
+        self.composite_logger.log_debug(
+            "Setting new installation reboot status. [NewRebootStatus={0}] [CurrentRebootStatus={1}]".format(
+                str(new_reboot_status), self.__installation_reboot_status))
```

Comment:
Please revert all PR noise like this. You can configure PyCharm to not prompt for these things. We have wider monitors than the terminal PyCharm is trying cater to. Vertical real estate is at a premium and reduces developer productivity when line breaks are used liberally in non-critical code.